tools/plasflow/test-data/output

```diff
@@ -0,0 +1,3 @@
+	contig_id	contig_name	contig_length	id	label	chromosome.Acidobacteria	chromosome.Actinobacteria	chromosome.Bacteroidetes	chromosome.Chlamydiae	chromosome.Chlorobi	chromosome.Chloroflexi	chromosome.Cyanobacteria	chromosome.DeinococcusThermus	chromosome.Firmicutes	chromosome.Fusobacteria	chromosome.Nitrospirae	chromosome.other	chromosome.Planctomycetes	chromosome.Proteobacteria	chromosome.Spirochaetes	chromosome.Tenericutes	chromosome.Thermotogae	chromosome.Verrucomicrobia	plasmid.Actinobacteria	plasmid.Bacteroidetes	plasmid.Chlamydiae	plasmid.Cyanobacteria	plasmid.DeinococcusThermus	plasmid.Firmicutes	plasmid.Fusobacteria	plasmid.other	plasmid.Proteobacteria	plasmid.Spirochaetes
```

Comment:
Deze file was alleen net wat anders qua afronding dan je originele test file, maar dat ligt vaak aan het OS, dus travis krijgt een antwoord eruit terwijl lokaal de afronding net wat anders gaat, deze tests worden ook gebruikt door Galaxy admins om hun geinstalleerde tools te testen, dus we willen niet falen op dit soort kleine verschillen. 

In dit soort gevallen is het vaak beter om te testen met [assert contents](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-output-assert-contents) en bijvoorbeeld checken dat de headerline in de file staat en het aantal kolommen klopt, en bepaalde belangrijke text (bijv `utg00144c` en `plasmid.Proteobacteria`) ..genoeg zodat je zeker weet dat de output klopt, maar dat kleine verschillen in afronding nog wel geaccepteerd worden ..voor voorbeelden van hoe dit te gebruiken kun je mijn mothur tools in tools-iuc repo bekijken btw